.travis.yml

```diff
@@ -0,0 +1,38 @@
+# Based on https://github.com/NOAA-ORR-ERD/gridded/blob/master/.travis.yml
+language: minimal
+
+sudo: false
+
+notifications:
+  email: false
+
+matrix:
+  fast_finish: true
+  include:
+    - name: "docs"
+    - name: "compass"
+    #- name: "atmosphere"
+    #- name: "landice"
+    #- name: "ocean"
+    #- name: "seaice"
+
+env:
+  global:
+    secure: "Ckovv5psUv9pdBVaNjsgrk0hXGJbNTi5dzLSWsNQA3Ub4hUXvzgBvX1TgS+CZvq3fkJnxV6SYZIlOkrCqqxdlUzFewDiuzzYxEZObFZ8FceWiYs4K3LMS30MPLzIYaV9ORqXJ0/P/ii62KhGIraX7ryzZ5+IG5uDOd7gumD6GiCPURFQIGpWS7MasKmEQRnokrGO3B2JjGEB0vMTdMZGbj8XY+X4Q0zcdQnjrLtG7PiuED95CrfEI7HfJ/ifPRgI4EMey6YI89DUF6hdQ1N1QxyAnxlKBDkqCvy6uowej7xPwMMZsyykr6EoAcwk9kYluVRogKe1iBMq1P4WCOOUj3c3Q9Dp885TiXlsW1aDKT2RSkx/SV5cE73vrpUhmm2Nf/wWAZktOYJSGlmRqSnjkQhDJqQSOaIQbwBfUtuGMt0Go0Q/3/qlMc7I9AjG5yBuYcY8WN98qKABP3IKjSsIFUPyo1kXCHt++oV7z1p9oBdGS54e+9rEYWixvAwvw0GvrNrbQdiq2sq8BheI4ppzeSGoneHguNDn5QyVeMu4WZ0gsCwuVW0lXiuUSlYHG5ZctX1cQySCyL3IOcpYa1a1/DKINpkwr/Z7T8oOM+UHnXDK96UJfbaG5n5+JZOfWJ6N/Em7weKuwnRiobGQfDqlyEtc1hgcLfvMnkpikENKhss="
```

Comment:
This lets Travis log in as me to upload the documentation to the `gh-pages` branch (once we create it).  This capability means that we need to be careful not to merge malicious code (e.g. "bug fixes" from an unknown contributor) that could use this ability to mess up the repo.  Just want to make that risk known so we're vigilant, though I think it is very unlikely.